packages/world-modules/test/SystemSwitch.t.sol

```diff
@@ -194,6 +212,20 @@ contract SystemSwitchTest is Test, GasReporter {
     assertEq(abi.decode(returnData, (string)), "hello");
   }
 
+  function testCallNonRootFromRootReadTable() public {
+    vm.prank(caller);
+
+    // Call reverts because the non-root system storage does not have table schemas
+    vm.expectRevert(
+      abi.encodeWithSelector(
+        IStoreErrors.Store_TableNotFound.selector,
+        ResourceIdsTableId,
+        string(abi.encodePacked(ResourceIdsTableId))
+      )
+    );
+    _executeFromRootSystemA(systemBId, abi.encodeCall(EchoSystem.readTable, ()));
```

Comment:
This isn't catching the revert properly:

<img width="679" alt="image" src="https://github.com/latticexyz/mud/assets/29184158/f27e7336-6bd8-4324-9b0e-d31d83170a45">

Removing it prints the error we want:
<img width="1469" alt="image" src="https://github.com/latticexyz/mud/assets/29184158/71bd5f95-6172-413c-8a81-d063ab256636">